backend/qemu.pm

```diff
@@ -418,8 +418,15 @@ sub start_qemu {
     my $qemubin = $ENV{QEMU};
 
     my $qemuimg = find_bin('/usr/bin/', qw(kvm-img qemu-img));
+
     unless ($qemubin) {
-        $qemubin = find_bin('/usr/bin/', $vars->{QEMU} ? ('qemu-system-' . $vars->{QEMU}) : qw(kvm qemu-kvm qemu qemu-system-x86_64 qemu-system-ppc64));
+        if ($vars->{QEMU}) {
+            $qemubin = find_bin('/usr/bin/', 'qemu-system-' . $vars->{QEMU});
+        }
+        else {
+            (my $class = $vars->{WORKER_CLASS} || '') =~ s/qemu_/qemu-system\-/g;
+            $qemubin = find_bin('/usr/bin/', (split(/\s*,\s*/, $class), qw(kvm qemu-kvm qemu qemu-system-x86_64 qemu-system-ppc64)));
```

Comment:
I have one more question: if WORKER_CLASS=tap,qemu_x86_64 will you start /usr/bin/tap?

Reply:
@coolo You do good questions :+1:  :-D

got the point.

Fixed